lib/codebreaker/hmi.rb

```diff
@@ -0,0 +1,52 @@
+module Codebreaker
+  class Hmi
+    def initialize
+      @game = Game.new
+    end
+
+    def launch_game
+      greeting
+      @game.start
+      until @game.game_over?
+        answ = gets.chomp
+        if answ =~ /^[1-6]{4}$/
+          @game.code_check(answ)
+          @game.show_attempt_result
+        elsif answ =~ /^[Hh]{1}$/
```

Comment:
`elsif`  not very good approach. What will be if count of commands increase to 10?
You can go in the direction of using the hash with the commands (kind of `{ command_name: command_method, ... }`) and resolver, which will check whether the code or command was entered